js/packages/offline-renderer/__tests__/offline-renderer.test.js

```diff
@@ -42,6 +42,10 @@ test('events', async function() {
   core.render(el.meter(el.mul(2, 3)));
   core.process(inps, outs);
 
+  // Block until next event loop iteration
+  // to wait for meter events
+  await new Promise(process.nextTick);
```

Comment:
The events are not propagated before checking the expectation, so this waits for them. That this is needed may indicate a subtle difference between `emittery` and node `events`. 

Does this impact the way users interact with the `offline-renderer`? Is it a breaking change?

Reply:
We have switched back to the `events` library for this PR.